tests/unit/components/Navigator.spec.js

```diff
@@ -6,7 +6,7 @@
  *
  * See https://swift.org/LICENSE.txt for license information
  * See https://swift.org/CONTRIBUTORS.txt for Swift project authors
-*/
+ */
```

Comment:
```suggestion
*/
```

(Probably auto modified by your editor)